src/model/beneficial.owner.other.model.ts

```diff
@@ -8,16 +8,21 @@ export const DateKey: string = "startDate";
 export const DateKeys: string[] = ["startDate-day", "startDate-month", "startDate-year"];
 
 export const BeneficialOwnerOtherKey: string = "beneficialOwnerOther";
-export const BeneficialOwnerOtherKeys: string[] = ["corporationName", "principalAddress", "isSameAddress", "serviceAddress", "lawGoverned", "startDate", "natureOfControl", "statementCondition", "isSanctioned"];
+export const BeneficialOwnerOtherKeys: string[] = ["corporationName", "principalAddress", "isSameAddress", "serviceAddress", "legalForm", "lawGoverned", "registerName", "registerNumber", "publicRegister", "startDate", "natureOfControlIndividual",  "natureOfControlTrust", "natureOfControlFirm", "isSanctioned"];
 
 export interface BeneficialOwnerOther {
-    corporationName?: string
-    principalAddress?: Address
-    isSameAddress?: yesNoResponse
-    serviceAddress?: Address
-    lawGoverned?: string
-    startDate?: InputDate
-    natureOfControl?: natureOfControl
-    statementCondition?: statementCondition
-    isSanctioned?: yesNoResponse
+    corporationName?: string;
```

Comment:
The model does not match what expected on the api and passed to the sdk

Reply:
No at the moment there is a subtask to deal with that: https://companieshouse.atlassian.net/browse/ROE-318

Reply:
the field names should match the api model names where possible like name, principal_address, service_address, is_service_address_same_as_principal_address
match the field names up with the field names from the existing api spec - 
{
      "name": "string",
      "principal_address": {
        "property_name_number": "string",
        "line_1": "string",
        "line_2": "string",
        "town": "string",
        "county": "string",
        "country": "string",
        "postcode": "string"
      },
      "service_address": {
        "property_name_number": "string",
        "line_1": "string",
        "line_2": "string",
        "town": "string",
        "county": "string",
        "country": "string",
        "postcode": "string"
      },
      "is_service_address_same_as_principal_address": true,
      "legal_form": "string",
      "law_governed": "string",
      "is_on_register_in_country_formed_in": true,
      "register_name": "string",
      "registration_number": "string",
      "start_date": ,
      "beneficial_owner_nature_of_control_types": [
        "over_25_percent_of_shares"
      ],
      "trustees_nature_of_control_types": [
        "over_25_percent_of_shares"
      ],
      "non_legal_firm_members_nature_of_control_types": [
        "over_25_percent_of_shares"
      ],
      "is_on_sanctions_list": true
    }